android/trifle/src/androidTest/java/app/cash/trifle/TrifleApiTest.kt

```diff
@@ -21,47 +21,48 @@ class TrifleApiTest {
   @Rule
   val thrown: ExpectedException = ExpectedException.none()
 
+  private val trifleApi = TrifleApi("app.cash.trifle.keys")
   private lateinit var keyHandle: KeyHandle
   @Before fun setUp() {
-    keyHandle = TrifleApi.generateKeyHandle("test-alias")
+    keyHandle = trifleApi.generateKeyHandle()
   }
 
   @Test fun testGenerateKeyHandle() {
```

Comment:
can we add a test to make sure a UUID is added to the reverse domain?

Reply:
Hmm kind of but what values are you proposing

Reply:
added a check if UUID is parseable